src/oas2/__tests__/bundle.test.ts

```diff
@@ -4,6 +4,147 @@ import * as path from 'path';
 import { bundleOas2Service } from '../service';
 
 describe('bundleOas2Service', () => {
+  it('should not bundle shared requestBodies', () => {
+    expect(
+      bundleOas2Service({
+        document: {
+          'x-stoplight': {
+            id: 'abc',
+          },
+          swagger: '2.0',
+          paths: {
+            '/users/{userId}': {
+              patch: {
+                parameters: [
+                  {
+                    $ref: '#/parameters/ids',
+                  },
+                  {
+                    $ref: '#/parameters/email',
+                  },
+                ],
+              },
+              put: {
+                parameters: [
+                  {
+                    $ref: '#/parameters/ids',
+                  },
+                ],
+              },
+              post: {
+                parameters: [
+                  {
+                    $ref: '#/parameters/email',
+                  },
+                ],
+              },
+            },
+          },
+          parameters: {
+            ids: {
+              in: 'formData',
+              name: 'ids',
+              type: 'array',
+              items: {
+                type: 'number',
+              },
+              required: true,
+              maxItems: 10,
+              minItems: 1,
+            },
+            email: {
+              in: 'body',
+              name: 'email',
+              required: true,
+              schema: {
+                format: 'e-mail',
+              },
+            },
+          },
+        },
+      }),
+    ).toStrictEqual({
+      id: 'abc',
+      name: 'no-title',
+      version: '',
+      operations: [
+        {
+          id: 'c911d0f53d100',
+          method: 'patch',
+          path: '/users/{userId}',
+          extensions: {},
+          request: {
+            body: {
+              id: '9b5d65cfbc4fd',
+              contents: [],
+              required: true,
+            },
```

Comment:
Hmm I'm a little confused - where is the info from those shared parameters? Haven't we lost information in this bundled object versus the original openapi spec?

Reply:
We don't bundle them, yeah, so we lost one

Reply:
The thing is we need `consumes` in order to build a request payload.
Unlike OAS 3.x, request bodies in v2 aren't "independent".

> There is no information in the output object about the request body

because I didn't add any `consumes`, that's how http-spec was apparently implemented work for OAS 2.0.
If there's no `consumes`, we don't do anything meaningful 🤷‍♂️

Reply:
Ah ok - so if there is a consumes at top level of openapi, or on the specific operation, the data would be for the most part preserved in the output?